src/components/Text.js

```diff
@@ -74,3 +74,5 @@ Text.defaultProps = defaultProps;
 Text.displayName = 'Text';
 
 export default Text;
+
+export {propTypes, defaultProps};
```

Comment:
Sorry, but we don't do it this way. 
here is the suggestion
1. Create `baseTextPropTypes.js` file and export {propTypes, defaultProps};. 
2. import as * from baseTextPropTypes on both Text and EmojiText components.
3. Place baseTextPropTypes in Text component folder.

Reply:
I have updated the pull request with new commits which takes care of this.
Thank you for your patience.